lesson07/assignment/recipes/src/components/ConversionsPage.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div id="app">
+    <div>
+      <h1 class="display-4 lead text-center">Common Conversions</h1>
+    </div>
+    <div class="accordion" :id="'accordionExample'+id" v-for="(conversion, id) in conversions" :key="id" >
+      <div class="card">
+          <div class="card-header" id="headingOne">
+              <h2 class="mb-0">
+                  <button class="btn btn-link text-dark text-decoration-none collapsed" type="button" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                    {{ conversion.title }} 
+                  </button>
+              </h2>
+          </div>
+
+          <div id="collapseOne" class="collapse" aria-labelledby="headingOne" :data-parent="'#accordionExample'+id">
```

Comment:
Since you aren't changing the id each time through the loop, clicking on accordion title toggles them all.  To fix this you can update the id here on line 1 and the data-target on line 10 to be dynamic based on the loop id.  So instead of collapseOne, you would use something like :id="collapse+id".